Backpack/src/main/java/net/skyscanner/backpack/dialog/DialogVariables.kt

```diff
@@ -0,0 +1,21 @@
+package net.skyscanner.backpack.dialog
+
+import android.content.Context
+import net.skyscanner.backpack.R
+
+internal class DialogVariables(val context: Context) {
+
+  private val spacingSm = context.resources.getDimension(R.dimen.bpkSpacingSm)
+
+  val iconContainerSize = (spacingSm * 18).toInt()
```

Comment:
use a constant directly. Changing spacing value will change the icons size which is not the expected behaviour

Reply:
This is the container around the icon and the icon itself, and this is based on the IOS version that does the same thing. Also there is no constant size (token) for this size.

Reply:
I still don't understand how would this cause a problem. Again this is only for the circular container around the icon and not the icon itself, it will not affect the icon size/density. Also we do the same with react native in a few cases and I don't think it is a problem. Can you explain how this would cause a problem? I'm sure we will need to do this more times as it is quite common in the RN components.

Reply:
The problem is if we update the spacing value, we will need to verify at every place where such maths is done instead of just the spacing instances. An icon size can not be 18 times spacing value, it gives you the correct value but does not make any sense, these are 2 independent things. While updating the RN codebase to apply lint rules, there were instances where I used calculations where it was logical and ignored the rule where it did not. Eg, you can achieve the action bar size of `56dp` by multiplying `spacingXs` by `56` but is not a logical thing to do.